src/app/globals.css

```diff
@@ -24,7 +24,15 @@ body {
   align-items: center;
   justify-content: center;
 }
+.flex {
+  display: flex;
+}
 
+.center{
```

Comment:
should not need to create a new class for this we could use the tailwind `flex justify-center align-center`